packages/notification-center-vue/package.json

```diff
@@ -22,7 +22,11 @@
     "@emotion/css": "^11.10.5",
     "@novu/notification-center": "^0.10.2",
     "floating-vue": "https://github.com/LetItRock/floating-vue/raw/main/packages/floating-vue/floating-vue-2.0.0-beta.21.tgz",
-    "vue": "^3.2.45"
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1"
```

Comment:
these should be defined as the direct dependencies, because in the `notification-center` they are defined as the `peer`